std/algorithm/comparison.d

```diff
@@ -560,7 +560,7 @@ if (isInputRange!R1 && isInputRange!R2 && !(isSomeString!R1 && isSomeString!R2))
     }
 }
 
-// Specialization for strings (for speed purposes)
+/// Specialization for strings (for speed purposes)
```

Comment:
Shouldn't this just be `/// ditto`? The docs for the previous overload of `cmp` already describes the behaviour with string arguments.